readme.md

```diff
@@ -159,6 +159,7 @@ Thanks for contributing! 🦋🙌
 - [](# "unreleased-commits") 🔥 [Tells you whether you're looking at the latest version of a repository, or if there are any unreleased commits.](https://user-images.githubusercontent.com/1402241/234576563-1a0ca255-4c0d-45ae-883d-2b1aa2d7f4c1.png)
 - [](# "action-pr-link") 🔥 [Adds a link back to the PR that ran the workflow.](https://github-production-user-asset-6210df.s3.amazonaws.com/50487467/241645264-076a0137-36a2-4fd0-a66e-735ef3b3a563.png)
 - [](# "mobile-tabs") [Makes the tabs more compact on mobile so more of them can be seen.](https://user-images.githubusercontent.com/1402241/245446231-28f44b59-0151-4986-8cb9-05b5645592d8.png)
+- [](# "repo-header-info") [Shows repository's stargazer count on the header.](https://github.com/refined-github/refined-github/assets/50487467/747b5133-61b8-490e-b961-4e5390ea9482)
```

Comment:
/assets/ URLs are not accepted by our parser, you'll have to open the link and get its real URL ending with .png (already done in a commit)

<img width="319" alt="Screenshot 13" src="https://github.com/refined-github/refined-github/assets/1402241/c117ff4a-133a-4a46-9f28-b27eb80b0fde">

Reply:
Also your screenshot was pixelated, make sure this doesn't happen in the future: 
![](https://github-production-user-asset-6210df.s3.amazonaws.com/50487467/266835037-747b5133-61b8-490e-b961-4e5390ea9482.png)